js/constants/apiUrls.js

```diff
@@ -0,0 +1,15 @@
+const mainUrl = 'https://bootcamp-api.codeit.kr/api/';
+
+const AUTH = {
+  SIGN_IN: `${mainUrl}sign-in`,
+  SIGN_UP: `${mainUrl}sign-up`,
+};
+
+const USER = {
+  CHECK_EMAIL: `${mainUrl}check-email`,
+};
```

Comment:
아래와 같이 사용하는게 가독성이 더 좋다고 생각하는데요~
mainUrl의 마지막 슬래시를 제거해주는게 좋을 것 같아요!
```suggestion
const AUTH = {
  SIGN_IN: `${mainUrl}/sign-in`,
  SIGN_UP: `${mainUrl}/sign-up`,
};

const USER = {
  CHECK_EMAIL: `${mainUrl}/check-email`,
};
```

Reply:
이렇게 api마다 url 경로를 상수화하는 것도 괜찮은데요~
오히려 이렇게 상수화를 하게 되면서 코드가 이해하기 어려워지지 않을까? 라는 고민도 해보면 좋을 것 같아요!